app/repositories/AnalyticsDataCache.scala

```diff
@@ -60,16 +68,26 @@ object AnalyticsDataCacheEntry {
 }
 
 object AnalyticsDataCache {
+
   def deleteCacheEntry(key: String, dataType: String): Unit = {
     Dynamo.analyticsDataCacheTable.deleteItem("key", key, "dataType", dataType)
   }
 
-
   def putCampaignDailyCountsReport(campaignId: String, data: CampaignDailyCountsReport, expires: Option[Long]): Unit = {
     val entry = AnalyticsDataCacheEntry(campaignId, "CampaignDailyCountsReport", Json.toJson(data).toString(), expires, System.currentTimeMillis())
     Dynamo.analyticsDataCacheTable.putItem(entry.toItem)
   }
 
+  def putCampaignSummary(campaignId: String, data: CampaignSummary, expires: Option[Long]): Unit = {
```

Comment:
Perhaps it should be clearer that `expires` is a time instant rather than an interval of time.